nsxt/resource_nsxt_policy_edge_transport_node.go

```diff
@@ -0,0 +1,1625 @@
+								Type:         schema.TypeString,
+								Optional:     true,
+								Description:  "Default IPv6 gateway for the node",
+								ValidateFunc: validateSingleIP(),
+							},
+							"management_port_subnet": {
+								Type:        schema.TypeList,
+								Required:    true,
+								MaxItems:    1,
+								MinItems:    1,
+								Description: "Ipv6 Port subnet for management port",
+								Elem: &schema.Resource{
+									Schema: map[string]*schema.Schema{
+										"ip_addresses": {
+											Type:        schema.TypeList,
+											Required:    true,
+											Description: "IPv6 Addresses",
+											MinItems:    1,
+											MaxItems:    1,
+											Elem: &schema.Schema{
+												Type:         schema.TypeString,
+												ValidateFunc: validateSingleIP(),
+											},
+										},
+										"prefix_length": {
+											Type:         schema.TypeInt,
+											Required:     true,
+											Description:  "Subnet Prefix Length",
+											ValidateFunc: validation.IntBetween(1, 128),
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+				"static_ipv6_list": {
+					Type:        schema.TypeList,
+					Description: "IP assignment specification value for Static IPv4 List",
+					Optional:    true,
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"default_gateway": {
+								Type:         schema.TypeString,
+								Required:     true,
+								Description:  "Gateway IP",
+								ValidateFunc: validateSingleIP(),
+							},
+							"ip_addresses": {
+								Type:        schema.TypeList,
+								Required:    true,
+								Description: "List of IPv6 IPs for edge transport node host switch virtual tunnel endpoints",
+								MaxItems:    32,
+								Elem: &schema.Schema{
+									Type:         schema.TypeString,
+									ValidateFunc: validateSingleIP(),
+								},
+							},
+							"prefix_length": {
+								Type:        schema.TypeInt,
+								Required:    true,
+								Description: "Prefix Length",
+							},
+						},
+					},
+				},
+				"static_ipv6_mac_list": {
+					Type:        schema.TypeList,
+					Description: "IP and MAC assignment specification for Static IPv6 List",
+					Optional:    true,
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"default_gateway": {
+								Type:         schema.TypeString,
+								Required:     true,
+								Description:  "Gateway IP",
+								ValidateFunc: validateSingleIP(),
+							},
+							"ip_mac_pair": getIpMacPairSchema(),
+							"prefix_length": {
+								Type:        schema.TypeInt,
+								Required:    true,
+								Description: "Prefix Length",
+							},
+						},
+					},
+				},
+				"static_ipv6_pool": {
+					Type:        schema.TypeString,
+					Description: "IP assignment specification for Static IPv6 Pool. Input can be MP ip pool UUID or policy path of IP pool",
+					Optional:    true,
```

Comment:
Should we enforce policy path here? No reason to use MP pool id

Reply:
Makes sense - the MP pool id capability is provided by NSX (the description is the API's)